src/highlight/HighlightList.scss

```diff
@@ -4,10 +4,9 @@
     left: 0;
     width: 100%;
     height: 100%;
-    mix-blend-mode: multiply;
 
-    &.is-listening {
-        .ba-HighlightAnnotation-rect {
+    .ba-HighlightSvg.is-listening {
```

Comment:
Is the `is-listening` code required at all? It was only necessary for regions in order to support highlighting, if memory serves.

Reply:
It's needed in order to be able to select text underneath the rendered HighlightTargets